pkg/notes/document/testdata/document.md.golden

```diff
@@ -1,7 +1,3 @@
-# Release notes for v1.16.1
-
-[Documentation](https://docs.k8s.io/docs/home)
```

Comment:
I would fix this link to https://docs.k8s.io and maybe add a better explanation like:

> The latest documentation for Kubernetes can be found on [docs.k8s.io](https://docs.k8s.io)

Reply:
I think this is meant to be removed because the original release notes generated by krel for patch releases doesn't have this section.

Or is it meant to be different?

Reply:
Ah, got it. :+1: